app/Bus/Commands/ComponentGroup/AddComponentGroupCommand.php

```diff
@@ -40,14 +40,28 @@
     public $collapsed;
 
     /**
+     * To whom should the component group be visible?
+     *
+     * @var int
+     */
+    public $visible;
```

Comment:
I think this is better named as `visibility` or something. To me, `visible` implies a boolean.